hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -107,24 +107,44 @@
                    "{tool}_otbn_tracer_build_opts",
                    "{tool}_memutil_dpi_scrambled_build_opts"]
 
+  // Setup for generating OTP images.
```

Comment:
just a note in this file, there seems to still be comments (not part of the change) that imply otp as another software image. 
Do you want to remove that also?

Reply:
Done.